src/main.cpp

```diff
@@ -505,13 +506,14 @@ int main(int argc, char *argv[])
         { "device",          required_argument, NULL, 'd' },
         { "log",             no_argument,       NULL, 'l' },
         { "audio",           optional_argument, NULL, 'a' },
+        { "to-yuv",          no_argument,       NULL, 't' },
         { 0,                 0,                 NULL,  0  }
     };
 
     int c, i;
     std::string param;
     size_t pos;
-    while((c = getopt_long(argc, argv, "o:f:g:c:p:d:la::", opts, &i)) != -1)
+    while((c = getopt_long(argc, argv, "o:f:g:c:p:d:lat::", opts, &i)) != -1)
```

Comment:
I just noticed this, shouldn't it be `la::t::`?

Reply:
I don't really know, you tell me? :)

Reply:
There, fixed.